autoload/maktaba/json.vim

```diff
@@ -12,6 +12,164 @@ if !exists('maktaba#json#NULL')
       \ 'null': s:NULL, 'true': s:TRUE, 'false': s:FALSE}
 endif
 
+" Python implementation:
+
+" Initialize Vim's Python environment with the helpers we'll need.
+function! s:InitPython() abort
+  python << EOF
```

Comment:
I think I may have to do that, yes. But it's actually not the import after all, but the call to `maktaba#GetJsonPythonDisabled()`, which triggers plugin loading (see above).